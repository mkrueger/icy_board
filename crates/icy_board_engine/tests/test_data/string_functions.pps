STRING S
INTEGER I, L

S = "*"
PRINTLN INSTR(S, "*")
S = "██*"
PRINTLN INSTR(S, "*")
PRINTLN INSTR(S, "A")

S = "F█o*B█r"
L = LEN(S)
PRINTLN L

I = INSTR(S, "*")
PRINT LEFT(S, I - 1)
PRINTLN MID(S, I + 1, L-I)
PRINTLN MID("12345", 3, 2)