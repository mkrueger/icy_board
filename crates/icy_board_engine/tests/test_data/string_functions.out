1
3
0
7
F█oB█r
34
FooBarBaz
